src/multi_stage/dual_dynamic_programming.jl

```diff
@@ -617,6 +617,11 @@ returns: JuMP model with updated objective function.
 """
 function initialize_cost_to_go(settings_d::Dict, EP::Model, inputs::Dict)
     cur_stage = settings_d["CurStage"] # Current DDP Investment Planning Stage
+    stage_len_1 = 0
```

Comment:
You could rename `stage_len_1` with something more descriptive, e.g.,: 
```suggestion
    cum_years = 0
```